### ISSUE TITLE
TextInput — Improve screen reader clarity for suggestions drop down

### ISSUE THREAD

**Issue by taysea** [grommet/grommet#7549]
TextInput suggestions doesn't provide a ton of clarity to screen reader users that there are suggestions available. This behavior should be improved.

Related to WCAG 1.3.1

It does a decent job after someone types, but it would be nice if it initially announced "X suggestions"

https://github.com/user-attachments/assets/d576a824-2e2d-406e-980f-a953ea045f4d

**Comment by taysea** [comment]
Upon further investigation, we are adhering to the guidelines from W3C in their authoring practices guide: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/#accessibilityfeatures

Closing this ticket..
